### PR TITLE
Fix code blocks in README

### PR DESCRIPTION
**Pull request by szepeviktor**
Please see comments below.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -38,7 +38,7 @@ Using it on [Bedrock](https://github.com/roots/bedrock) or [Sage](https://github
 ## Installation
 
 ```sh-session
-$ composer require --dev itinerisltd/itineris-wp-coding-standards
+composer require --dev itinerisltd/itineris-wp-coding-standards
```
Comment (szepeviktor) [inline README.md:41]: Make this code block copyable.

image

File: README.md [hunk 2]
```diff
@@ -77,16 +77,20 @@ First, create [`phpcs.xml`](https://github.com/squizlabs/PHP_CodeSniffer/wiki/An
     <!-- TODO: Define expected text domains -->
     <rule ref="WordPress.WP.I18n">
         <properties>
-            <property name="text_domain" type="array" value="my-plugin,my-theme,woocommerce,sage"/>
+            <property name="text_domain" type="array">
+                <element value="my-plugin"/>
+                <element value="my-theme"/>
+                <element value="woocommerce"/>
+                <element value="sage"/>
+            </property>
```
Comment (szepeviktor) [inline README.md:85]: Use modern PHPCS syntax for arrays.

File: README.md [hunk 3]
```diff
@@ -77,16 +77,20 @@ First, create [`phpcs.xml`](https://github.com/squizlabs/PHP_CodeSniffer/wiki/An
...
         </properties>
     </rule>
 </ruleset>
 ```
 
-Then, define [composer scripts](https://getcomposer.org/doc/articles/scripts.md):
+Then, define [composer scripts](https://getcomposer.org/doc/articles/scripts.md) in `composer.json`:
 
 ```json
-# composer.json
```
Comment (szepeviktor) [inline README.md:89]: Fix invalid JSON.
JSON and XML are not for humans.

**Review by codepuncher — APPROVED** [review]
Thanks @szepeviktor!

**Pull request merged**